lib/extend/renderer.ts

```diff
@@ -57,7 +59,7 @@ class Renderer {
     this.storeSync = {};
   }
 
-  list(sync: boolean): Store | SyncStore {
+  list(sync?: boolean): Store | SyncStore {
```

Comment:
```suggestion
  list(sync = false): Store | SyncStore {
```

Let's use default boolean value here.